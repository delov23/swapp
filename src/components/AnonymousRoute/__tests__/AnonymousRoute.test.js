import React from 'react';
import { shallow } from 'enzyme';

import { AnonymousRoute } from '../';

describe('<AnonymousRoute />', () => {
  it('should match snapshot', () => {
    expect(shallow(<AnonymousRoute />)).toMatchSnapshot();
  });
});
