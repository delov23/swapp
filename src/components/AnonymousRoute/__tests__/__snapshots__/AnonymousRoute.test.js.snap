// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnonymousRoute /> should match snapshot 1`] = `
<PermissionRoute
  permission="anonymous"
/>
`;
